packages/embark-ui/src/components/Accounts.js

```diff
@@ -2,17 +2,35 @@ import React from 'react';
 import {Row, Col, Card, CardHeader, CardBody} from 'reactstrap';
 import {Link} from 'react-router-dom';
 import PropTypes from 'prop-types';
+import Pagination from './Pagination';
 
 import CardTitleIdenticon from './CardTitleIdenticon';
 
-const Accounts = ({accounts}) => (
+function displayTruncatedBalance (balance) {
```

Comment:
Really cool function, though if you wanted, I think you could have went with a simpler implementation where you check if the number is higher than X, if so, just show `X+` and then you just round the decimals (because tbh, wei amounts are not that useful).
Anyway, you already did a good implementation here.

Reply:
Thanks!  Good point re: the possibility of something simpler, will keep that in mind next time I need a similar function.